tests/testEdges.py

```diff
@@ -367,8 +366,6 @@ def myimshow(*args, **kwargs):
             print('Wrote', fn)
 
 
-#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
```

Comment:
Yes, these should have been stripped out during the py.test port. This one must have been missed.